pallets/parachain-staking/src/mock.rs

```diff
@@ -125,6 +126,15 @@ parameter_types! {
 	pub const MinDelegation: u128 = 3;
 	pub const MaxCandidates: u32 = 200;
 }
+
+pub struct StakingRoundSlotProvider;
+impl Get<Slot> for StakingRoundSlotProvider {
+	fn get() -> Slot {
+		let block_number: u64 = System::block_number().into();
```

Comment:
I know this is a mock, so maybe it doesn't matter, but I do wonder if using block number here is conceptually fine. There's a bit of a difference right? Block number is the block on the parachain and the slot you want is the one for the relay chain. Maybe a raw u64 that monotonically increases is conceptually better than getting the parachain block? Am I missing something?

Reply:
Yeah that could be an option, but so far I think we can configure the mock like this, as this is still the case on Moonbeam and Moonriver. Once we enable async backing on these two other networks, we can adapt the mock for it to use the relay slot. The problem with the monotonically u64 is that in the case of the relay slot, it comes with a diff of two, so we would have a discrepancy there.